src/main/resources/db/migration/V1_20__add_sequence_to_description_addendum.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE if exists description_addendum;
```

Comment:
We will lose data temporarily

Reply:
but it's easier to recreate table than manually run sql as part of deployment and we gotta migrate anyway